src/FakeItEasy/FakeItEasy.nuspec

```diff
@@ -13,10 +13,41 @@
     <licenseUrl>https://github.com/FakeItEasy/FakeItEasy/blob/master/License.txt</licenseUrl>
     <copyright>Copyright (c) FakeItEasy contributors. (fakeiteasyfx@gmail.com)</copyright>
     <tags>TDD unittesting mocks mocking fakes faking stubs stubbing spy spies doubles isolation substitutes substitution</tags>
+    <dependencies>
+      <group targetFramework=".NETFramework4.0">
+        <dependency id="Newtonsoft.Json" version="9.0.1" />

```

Comment:
Yeah, that's the only usage, @jeremymeng.
@thomaslevesque, I agree that nearly nobody would miss it. On public GitHub (which is not nearly the world), only https://github.com/omnibs/service-ghost uses the feature.

I think we had  discussion about the feature in the run-up to 2.0 and decided to keep it. While one could argue that we're still not "removing" a feature, since we could never use `RecordedBy` on .NET Core before, find it a weak arguments. I'm uncomfortable at the idea of dropping features in the newer frameworks.
I'd rather introduce the dependency (or find another way to serialize) than lose the feature. Unless we decide to move to FIE 3.0 and drop it entirely…